examples/src/test/java/io/grpc/examples/routeguide/RouteGuideClientTest.java

```diff
@@ -96,16 +95,11 @@ public void setUp() throws Exception {
     // Use a mutable service registry for later registering the service impl for each test case.
     grpcCleanup.register(InProcessServerBuilder.forName(serverName)
         .fallbackHandlerRegistry(serviceRegistry).directExecutor().build().start());
-    client =
-        new RouteGuideClient(InProcessChannelBuilder.forName(serverName).directExecutor());
+    client = new RouteGuideClient(grpcCleanup.register(
```

Comment:
nit: I would declare a local variable for channel as this expression is somewhat complex.

Reply:
I considered that as well, but when I tried it it didn't really seem to help much. I think the best approach might be to make the channel/server fields which would remove most of the code from this `@Before`. But even that I'm not 100% sold on. I'll leave this as-is for now.